### PR TITLE
New ILP packet schema

### PR DESCRIPTION
**Pull request by justmoon**

### CODE REVIEW

File: schemas/IlpHeader.json [hunk 1]
```diff
@@ -5,25 +5,18 @@
...
-      "$ref": "Amount.json"
-    },
-    "expiresAt": {
-      "description": "Expiry timestamp",
-      "$ref": "Date.json"
-    },
-    "executionCondition": {
-      "description": "Crypto condition triggering execution of the payment",
-      "$ref": "Condition.json"
+      "description": "Positive integer amount (UInt64)"
```
Comment (sentientwaffle) [inline schemas/IlpHeader.json]: Missing a comma here

File: schemas/IlpHeader.json [hunk 2]
```diff
@@ -5,25 +5,18 @@
...
     "data": {
-      "description": "User-specified memo, will be the memo of the final transfer",
-      "type": "object",
-      "additionalProperties": true
+      "description": "User-specified memo to be delivered with the payment",
+      "type": "string",
+      "$ref": "Base64URL.json"
     }
   },
   "additionalProperties": false
```
Comment (sentientwaffle) [inline schemas/IlpHeader.json]: Does there need to be a `"required"` property?

Comment (justmoon) [inline schemas/IlpHeader.json]: Does `required` accept an empty string? (Empty string should be allowed.)

File: schemas/IlpHeader.json [hunk 3]
```diff
@@ -5,25 +5,18 @@
...
+      "description": "Positive integer amount (UInt64)"
+      "type": "string",
+      "pattern": "^[1-9][0-9]*$"
     },
     "data": {
-      "description": "User-specified memo, will be the memo of the final transfer",
-      "type": "object",
-      "additionalProperties": true
+      "description": "User-specified memo to be delivered with the payment",
+      "type": "string",
```
Comment (sentientwaffle) [inline schemas/IlpHeader.json]: `"type": "string"` is redundant here, because of `"$ref": "Base64URL.json"`.

File: schemas/IlpHeader.json (hunk 2 above)
Comment (sentientwaffle) [inline schemas/IlpHeader.json]: If the field is just `"type": "string"` then sure, but all of these fields have additional constraints or patterns

**Comment by justmoon** [comment]
@sentientwaffle Addressed your comments.

**Comment by justmoon** [comment]
Merged as part of #183

**Pull request closed without merging**